src/components/ImportRecordsFlow/stepTwo/index.js

```diff
@@ -68,8 +72,23 @@ export default function StepTwo(props) {
             onDrop={handleDrop}
             draggable
         >
-            <UploadFileButton onChange={handleChange} />
-            <StyledText>Find the file in your desktop.</StyledText>
+            <StyledUploadFileWrapper isDragOver={isDragOver}>
+                <UploadFileButton onChange={handleChange} />
+            </StyledUploadFileWrapper>
```

Comment:
if you want to hide UploadFileButton when isDragOver you can pass styles directly inside UploadFileButton or simply use a RenderIf, there is no need to create an extra wrapper

Reply:
that will be better